pkgs/development/python-modules/fountains/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, bitlist
+}:
+
+buildPythonPackage rec {
+  pname = "fountains";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0jk5y099g6ggaq5lwp0jlg4asyhcdxnl3him3ibmzc1k9nnknp30";
+  };
+
+  propagatedBuildInputs = [
+    bitlist
+  ];
+
+  # There are no tests at the moment only the requirement in tests_require
+  postPatch = ''
+    sed -i "/tests_require=["nose"],/d" setup.py
+  '';
```

Comment:
Why patch this if we set `doCheck = false` anyway?

Reply:
I guess that the same approach as for the coverage removal is unnecessary. Will leave `nose` alone.